pyramid_closure/views.py

```diff
@@ -7,7 +7,7 @@
 
 def pairwise(iterable):
     if isinstance(iterable, dict):
-        return iterable
+        return iterable.items()
```

Comment:
`izip` returns an iterator, so, for consistency, I'd rather use `iteritems()` here. I know Python 3 doesn't have `iteritems`, but we should use `six` if we want to be compatible with both Python 2 and 3.